account_payment_credit_card/views/account_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
```

Comment:
Not required, yet adding copyright & license to XML (which is kind-a code) would be great

Reply:
@alexey-pelykh @bodedra  No need to here, In XML file not mandatory.

Reply:
@nikul-serpentcs yes. it's not mandatory. But I have improve it at https://github.com/nikul-serpentcs/account-payment/pull/1